lib/samples/add_feature_layers/add_feature_layers_sample.dart

```diff
@@ -60,13 +65,16 @@ class _AddFeatureLayersSampleState extends State<AddFeatureLayersSample> {
   Widget build(BuildContext context) {
     return Scaffold(
       body: SafeArea(
```

Comment:
Is this another one of those scenarios where we want to turn off the SafeArea's "top" so that the map runs all the way up under the notch and what-not?

Reply:
I can add it but it doesn't seem cause any visual change. Is it just for rendering speed?

Reply:
It doesn't make a difference in the Sample Viewer. But! When you run the sample by itself in sample_runner.dart, it'll make a difference. The Sample Viewer adds an AppBar at the top, but sample_runner.dart does not. When there's no AppBar, we want the map to run all the way up to the top edge.

Reply:
Ohhh yes, I have seen this issue come up before, thanks!